REST-SDK/src/Cumulocity.SDK.Client/RestConnector.cs

```diff
@@ -76,22 +82,39 @@ public T postText<T>(string path, string content, Type representation)
 			return ResponseParser.parse<T>(response.Result, representation, (int)HttpStatusCode.Created, (int)HttpStatusCode.OK);
 		}
 
-		public T Post<T>(string path, CumulocityMediaType mediaType, T representation)
+		public virtual T Post<T>(string path, CumulocityMediaType mediaType, T representation)
 			where T : IResourceRepresentation
 		{
-			var response = httpPost(path, mediaType, mediaType, representation);
+			try
+            {
+				var response = httpPost(path, mediaType, mediaType, representation);
+
+				if (!PlatformParameters.requireResponseBody())
+				{
+					var repFromPlatform = ResponseParser.parse<T>(response.Result, typeof(T), (int)HttpStatusCode.Created, (int)HttpStatusCode.OK);
+					return repFromPlatform != null ? repFromPlatform : representation;
+				}
+
+				//if (((IList)typeof(T).GetInterfaces()).Contains(typeof(IBaseResourceRepresentationWithId)))
+				//{
```

Comment:
Is it required to maintain.If YES please make appropriate comments otherwise clean it up